extensions/ql-vscode/src/common/unzip.ts

```diff
@@ -0,0 +1,84 @@
+import { Entry as ZipEntry, open, Options as ZipOptions, ZipFile } from "yauzl";
+import { Readable } from "stream";
+
```

Comment:
What do you think about adding some tests for these functions? Using actual zip files we have in a test data dir.

Reply:
Good idea, I'll add those tests in a follow-up PR so I can continue working with these functions in other follow-up PRs.